### ISSUE TITLE
Video Optimization: VideoPress / Jetpack Integration

### ISSUE THREAD

**Issue by swissspidy** [ampproject/amp-wp#3045]
As discussed in #2984, we are going with a VideoPress integration in a first  step, and will build on top of such an integration to support other solutions as well.

https://github.com/ampproject/amp-wp/issues/2984#issuecomment-519672995 mentions a recommended approach, although I am not sure how much of that applies to VideoPress; given that its implementation seems to already work automatically behind the scenes.

However, there currently seems to be an issue in Jetpack that prevents videos uploaded via the video block from being optimized (see https://github.com/Automattic/jetpack/issues/11194, https://jetpack.com/support/jetpack-blocks/video-block/, and https://jetpack.com/support/video-hosting/).

**Comment by spacedmonkey** [comment]
To start work on this, the is required. 

- A jetpack account with a paid for subscription that supports videopress. 
- Add Jetpack plugin to testing environment, activate it and setup VideoPress. 
- Find someway to use videopress locally in a development environment. 

 I believe this issue is blocked by this Jetpack upstream PR https://github.com/Automattic/jetpack/pull/13395 which doesn't seem to be moving. Until Jetpack is fixed, I recommend moving this out of sprint 14 / 15 and move it to the backlog. Thoughts @MackenzieHartung ?

**Comment by spacedmonkey** [comment]
Interesting and related is this news - https://rtmedia.io/blog/transcoder-for-amp-video-stories

Not heard of rtmedia personally before this was flagged in the slack channel. Plans for this service start from free to $9 a month, which seems pretty reasonable. Plugin already integrated with AMP plugn 1.3+.

**Comment by MackenzieHartung** [comment]
On hold given the rtmedia transcoder option.

**Comment by swissspidy** [comment]
Some good news: https://github.com/Automattic/jetpack/pull/13682 has been merged, which means Jetpack 7.9 will have better VideoPress integration 💪 🎉
